packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -106,40 +178,49 @@ export default function EntitiesSavedStates( { isOpen, onRequestClose } ) {
 			saveEditedEntityRecord( kind, name, key );
 		} );
 
-		onRequestClose( entitiesToSave );
+		closePanel( entitiesToSave );
 	};
 
-	return (
-		isOpen && (
-			<Modal
-				title={ __( 'What do you want to save?' ) }
-				onRequestClose={ () => onRequestClose() }
-				contentLabel={ __( 'Select items to save.' ) }
-			>
-				{ partitionedSavables.map( ( list ) => {
-					return (
-						<EntityTypeList
-							key={ list[ 0 ].name }
-							list={ list }
-							unselectedEntities={ unselectedEntities }
-							setUnselectedEntities={ setUnselectedEntities }
-						/>
-					);
-				} ) }
+	const dismissPanel = useCallback( () => closePanel(), [] );
```

Comment:
Redundant `useCallback`.

Reply:
I think this is still necessary in that in the post editor `closePanel` fires the callback added in the publish button.

If we aren't specific about firing it with no arguments, then the event object will be passed in where we would expect to see `savedEntities` and cause an error.  I suppose we could define this without using `useCallback` since the dependency array is empty, but doesn't `useCallback` improve performance via memoization here anyways?

Reply:
Ahh, that makes sense. Let's add a comment to make it clear.